### PR TITLE
Fix night shading in 2D

### LINKED ISSUE #4122: Day/Night lighting broken in 2D
It only 
orks sometimes depending on where the camera is
day-night-2d

``` javascript
viewer.scene.globe.enableLighting = true;
```

### PR DESCRIPTION
**Pull request by bagnell**
Fixes #4122.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @bagnell!
* :heavy_check_mark: Signed CLA found.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

File: CHANGES.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Change Log
 * The Viewer widget now takes terrain altitude into account when zooming or flying to imagery layers.
 * Fixed bug that caused a new `ClippingPlaneCollection` to be created every frame when used with a model entity [#6872](https://github.com/AnalyticalGraphicsInc/cesium/pull/6872)
 * Fixed crash when rendering translucent objects when all shadow maps in the scene set `fromLightSource` to false. [#6883](https://github.com/AnalyticalGraphicsInc/cesium/pull/6883)
+* Fixed night shading in 2D. [#4122](https://github.com/AnalyticalGraphicsInc/cesium/issues/4122)
```
Comment (mramato) [inline CHANGES.md]: And Columbus View?

**Comment by hpinkos** [comment]
Thanks @bagnell!

File: CHANGES.md (hunk 1 above)
Comment (mramato) [reply CHANGES.md]: Nevermind, didn't realize it already worked there.

**Pull request merged**